charts/images.yaml

```diff
@@ -204,34 +204,107 @@ images:
   sourceRepository: github.com/fluent/fluent-bit
   repository: fluent/fluent-bit
   tag: "1.9.7"
+  labels:
+  - name: 'gardener.cloud/cve-categorisation'
+    value:
+      network_exposure: 'private'
+      authentication_enforced: false
+      user_interaction: 'gardener-operator'
+      confidentiality_requirement: 'high'
+      integrity_requirement: 'low'
+      availability_requirement: 'low'
 - name: fluent-bit-plugin-installer
   sourceRepository: github.com/gardener/logging
   repository: eu.gcr.io/gardener-project/gardener/fluent-bit-to-loki
   tag: "v0.47.0"
+  labels:
+  - name: 'gardener.cloud/cve-categorisation'
+    value:
+      network_exposure: 'private'
+      authentication_enforced: false
+      user_interaction: 'gardener-operator'
+      confidentiality_requirement: 'high'
+      integrity_requirement: 'low'
+      availability_requirement: 'low'
+      comment: this should be either the same as fluent-bit or just skipped because the image itself is only a carrier.
 - name: loki
   sourceRepository: github.com/grafana/loki
   repository: grafana/loki
   tag: "2.2.1"
+  labels:
+  - name: 'gardener.cloud/cve-categorisation'
+    value:
+      network_exposure: 'private'
+      authentication_enforced: false
+      user_interaction: 'gardener-operator'
+      confidentiality_requirement: 'high'
+      integrity_requirement: 'low'
+      availability_requirement: 'low'
 - name: loki-curator
   sourceRepository: github.com/gardener/logging
   repository: eu.gcr.io/gardener-project/gardener/loki-curator
   tag: "v0.47.0"
+  labels:
+  - name: 'gardener.cloud/cve-categorisation'
+    value:
+      network_exposure: 'private'
+      authentication_enforced: false
+      user_interaction: 'gardener-operator'
+      confidentiality_requirement: 'none'
+      integrity_requirement: 'low'
+      availability_requirement: 'none'
 - name: kube-rbac-proxy
   sourceRepository: github.com/brancz/kube-rbac-proxy
   repository: quay.io/brancz/kube-rbac-proxy
   tag: v0.14.0
+  labels:
+  - name: 'gardener.cloud/cve-categorisation'
+    value:
+      network_exposure: 'public'
+      authentication_enforced: true
+      user_interaction: 'end-user'
+      confidentiality_requirement: 'none'
+      integrity_requirement: 'low'
+      availability_requirement: 'low'
+      comment: the network_exposure maybe should be considered protected because we restrict the request only to the push API.
```

Comment:
`network_exposure: 'public'` looks correct as long as there is interaction with peers from public internet. The restriction of the HTTP request method to `push` does not change the exposure.
<hr>

I have a question about  `user_interaction: 'end-user'`: 

> Declaring user interaction as end-user indicates that also untrained users can interact w/ the deployed service.

Do we really expect (and allow) untrained users to work with this component?
<hr>

Before removing the draft status of the PR, we shall update the comment labels replacing the questions with some explanations for our clasification decisions.

Reply:
> Do we really expect (and allow) untrained users to work with this component?
The promtail agent is on the end-user node. As they can have admins and ssh to their nodes they can manipulate the agent as they wish.